airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -2062,6 +2062,24 @@ definitions:
       type:
         type: string
         enum: [XmlDecoder]
+  CsvDecoder:
```

Comment:
This looks good, but you also need to add `CsvDecoder` as an option to all the spots in the schema that accept `XmlDecoder` as an option.